src/App.js

```diff
@@ -1,15 +1,43 @@
 import "./App.css";
-import { Fragment } from "react";
+import react, { Fragment, useEffect } from "react";
 import AddTodoForm from "./components/AddTodoForm";
 import TodoList from "./components/TodoList";
 import { useState } from "react";
```

Comment:
A small suggestion here: `useState ` can be moved to line 2 to save some typing. So it will end up: 
`import react, { Fragment, useEffect, useState } from "react";` on line 2 and line 5 removed.

Reply:
Solved.